conclave-init/src/main/kotlin/com/r3/conclave/init/cli/Main.kt

```diff
@@ -28,6 +28,7 @@ fun main(args: Array<String>) {
     headerHeading = "%n",
     synopsisHeading = "%nUsage:%n",
     optionListHeading = "%n",
+    versionProvider = ConclaveInitCli.ManifestVersionProvider::class,
```

Comment:
Question, as I don't know about this. That does `versionProvider` add practically? I.e. how is this used? Is there any convention or some automatic behaviour around?
Might be good to add description to the PR.

Reply:
It's required by picocli when retrieving version number dynamically (e.g. not from a const val).
I will now play with your suggestion using conclave version as a const variable as that would make it easier.

Reply:
I have added comment to the code. versionProvider is still required.